### ISSUE TITLE
Triplet selecting in tripletloss training

### ISSUE THREAD

**Issue by liujingqwq** [davidsandberg/facenet#435]
Hi, @davidsandberg 

When I am training my model using tripletloss_train.py , I  notice that, in this file,
 you provide two kinds of methods to select the negative examples :

1. all_neg = np.where(np.logical_and(neg_dists_sqr-pos_dist_sqr<alpha, pos_dist_sqr<neg_dists_sqr))[0]  #FaceNet selection
2. all_neg = np.where(neg_dists_sqr-pos_dist_sqr<alpha)[0] # VGG Face selecction

I would like to know : 
1.which method do you use to train the 511 and 512 model
2.when you use the VGG Face selecction ,will the negatives be too difficult to cause the model  collapsing

thanks very much

**Comment by davidsandberg** [comment]
All pretrained model so far have been trained using softmax loss.

**Comment by hungnv21292** [comment]
hi @davidsandberg 
I am training by triplet loss method. Could you please give me a suggestion how to choose triplet method?
Should I choose FaceNet selection or VGGFace selection?

Thank you very much.
